manual/src/cmds/intf-c.etex

```diff
@@ -2404,6 +2404,11 @@ The corresponding C type must be "intnat".
 {\bf Note:} Do not use the C "int" type in correspondence with "(int
 [\@untagged])". This is because they often differ in size.
 
+It is possible to annotate with "[\@untagged]" any immediate type, i.e. types
+that are represented like "int". This includes "bool", "char", any variant
+type with only constant constructors. Note: this does not include
+"Unix.file_descr" which is not represented as an interger on all platforms.
```

Comment:
I'm not sure if the note is necessary (in particular it is semi-clear from the fact that `Unix.file_descr` is not marked immediate), but let's grant this as a freedom of the person actually writing the documentation.